app/javascript/packages/document-capture/components/acuant-capture.jsx

```diff
@@ -125,6 +125,32 @@ function getDocumentTypeLabel(documentType) {
   }
 }
 
+/**
+ * @param {import('./acuant-capture-canvas').AcuantCaptureFailureError} error
+ *
+ * @return {string}
+ */
+export function getNormalizedAcuantCaptureFailureMessage(error) {
+  if (error instanceof window.MediaStreamError) {
```

Comment:
Testing this live, I don't think `MediaStreamError` is actually something exposed on the `window` object. We should be able to infer the same by just testing against `instanceof Error`. Will need a couple related revisions.

```suggestion
  if (error instanceof Error) {
```

Reply:
> Testing this live, I don't think `MediaStreamError` is actually something exposed on the `window` object. We should be able to infer the same by just testing against `instanceof Error`. Will need a couple related revisions.

Fixed in 156156d.